layers/geolocation/packages.el

```diff
@@ -13,32 +13,62 @@
 (setq geolocation-packages
     '(
       osx-location
+      popwin
+      rase
       sunshine
-      theme-changer
       ))
 
 (defun geolocation/init-osx-location ()
   "Initialize osx-location"
   (use-package osx-location
-    :if geolocation-enable-osx-location-service-support
+    :if geolocation-enable-location-service
+    :defer t
+    :init
+    (progn
+      (when (spacemacs/system-is-mac)
+          (add-hook 'osx-location-changed-hook
+                    (lambda ()
+                      (let ((location-changed-p nil)
+                            (_longitude (/ (truncate (* osx-location-longitude 10)) 10.0)) ; one decimal point, no rounding
+                            (_latitdue (/ (truncate (* osx-location-latitude 10)) 10.0)))
+                        (unless (equal calendar-longitude _longitude)
```

Comment:
I don't think that you can expect `calendar-longitude` being defined at this place.  `osx-location-changed-hook` might run before `calendar.el` is loaded.